chapters/operatorsandexpressions.tex

```diff
@@ -438,10 +436,10 @@ \subsection{Numeric Functions and Conversion Functions}\doublelabel{numeric-func
 
 Format string: According to ANSI-C the format string specifies one
 conversion specifier (excluding the leading \%), may not contain length
-modifiers, and may not use "\lstinline!*!" for width and/or precision. For~all
-numeric values~the~format specifiers~\lstinline!f!, \lstinline!e!, \lstinline!E!, \lstinline!g!,\lstinline!G! are allowed. For
-integral values~it is also allowed to~use the~d, i, o, x, X, u, and
-c-format specifiers (for non-integral values a tool may~round, truncate
+modifiers, and may not use `\lstinline!*!' for width and/or precision. For all
```

Comment:
I think this should be \lstinline!"*"! as the star occurs inside a C/Modelica string quoted with "".

Reply:
To me, this is the same as the redundant escapes.  Ideally, it shouldn't need quoting at all, but as this makes it too hard to separate from the surrounding text, single quotes are used outside of the `\lstinline`.